test/run_test.py

```diff
@@ -1122,21 +1098,19 @@ def load_specified_test_cases(filename: str) -> None:
     with open(filename, mode='r', encoding="utf-8-sig") as csv_file:
         csv_reader = csv.DictReader(csv_file)
         line_count = 0
-        global SPECIFIED_TEST_CASES_DICT
+        global SPECIFIED_TEST_MODULES
         for row in csv_reader:
             line_count += 1
             if line_count == 1:
                 if 'test_filename' not in row or 'test_case_name' not in row:
```

Comment:
I think the check for 'test_case_name' should then be moved to common_utils to minimize confusion